src/main/kotlin/com/zepben/evolve/database/sqlite/cim/tables/associations/TableSynchronousMachineReactiveCapabilityCurves.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2024 Zeppelin Bend Pty Ltd
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package com.zepben.evolve.database.sqlite.cim.tables.associations
+
+import com.zepben.evolve.database.sqlite.cim.tables.Column
+import com.zepben.evolve.database.sqlite.cim.tables.Column.Nullable.NOT_NULL
+import com.zepben.evolve.database.sqlite.cim.tables.SqliteTable
+
+@Suppress("PropertyName")
+class TableSynchronousMachineReactiveCapabilityCurves : SqliteTable() {
```

Comment:
todo: Missing doc strings (which lots of our tables are). See `TableCustomers` for examples. This applies to all new tables, and should also be added to the ones you have modified.

Reply:
Added to new table and all the association ones

Reply:
Class docstrings shouldn't have a blank line between the docstring and the class (or annotations). Can you please update this on all new class level docstrings you added/modified.

Reply:
All fixed up